docs/rabbitmq.config.example

```diff
@@ -120,12 +124,17 @@
    %%
    %% To use the SSL cert's CN instead of its DN as the username
    %%
-   %% {ssl_cert_login_from, common_name},
+   %% {ssl_cert_login_from, distinguished_name},
 
    %% SSL handshake timeout, in milliseconds.
    %%
    %% {ssl_handshake_timeout, 5000},
 
+   %% Whether to allow SSL POODLE attack,
```

Comment:
This makes it sound like anyone wants to enable an attack.

"Makes RabbitMQ accept SSLv3 client connections by default. DO NOT DO THIS IF YOU CAN HELP IT."